### PR TITLE
[APO] Adding content types

### PR DESCRIPTION
**Pull request by dcpena**
Added PCX content types to the APO docs

### CODE REVIEW

File: products/automatic-platform-optimization/src/content/about/index.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: About
 order: 1
+pcx-content-type: landing-page
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/about/index.md]: We've been designating this as a `concept` in most places.

File: products/automatic-platform-optimization/src/content/about/test-current-speed.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Test current speed
 order: 2
+pcx-content-type: tutorial
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/about/test-current-speed.md]: I'd say this is more of a `how-to`.

File: products/automatic-platform-optimization/src/content/about/plugin-compatibility.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Plugin compatibility
 order: 3
+pcx-content-type: tutorial
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/about/plugin-compatibility.md]: More of a `reference`

File: products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Activate the Cloudflare WordPress plugin
 order: 7
+pcx-content-type: how-to
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md]: This may be a `tutorial` since it's chaining together a lot of steps

File: products/automatic-platform-optimization/src/content/get-started/verify-apo-works.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Verify APO works
 order: 8
+pcx-content-type: tutorial
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/get-started/verify-apo-works.md:4]: I'd lean slightly towards `how-to` on this one, but I could also see tutorial

File: products/automatic-platform-optimization/src/content/reference/subdomain-subdirectories.md [hunk 6]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Subdomains and subdirectories
 order: 13
+pcx-content-type: tutorial
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/reference/subdomain-subdirectories.md]: Are these steps all related? Or just different ways to approach different scenarios?

If the former, `tutorial` works. If not, I'd think a `how-to`.

File: products/automatic-platform-optimization/src/content/troubleshooting/index.md [hunk 7]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Troubleshooting
 order: 14
+pcx-content-type: best-practices
 ---
 
 # Troubleshooting
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/troubleshooting/index.md:7]: To me, it feels like this might be another sorta `faq` page, but maybe that means you'd want to label as `interim`?

Like, I could see you combining this with the `FAQ` page?

**Review by kodster28 — APPROVED** [review]

File: products/automatic-platform-optimization/src/content/about/test-current-speed.md (hunk 2 above)
Comment (dcpena) [reply products/automatic-platform-optimization/src/content/about/test-current-speed.md]: Would this one actually be more of a tutorial? Because the how-to's purpose is to explain tasks within the product, but this isn't part of the product. I thought this kinda fit in more with the real-world scenario purpose of the tutorial because users would be learning how their site performs for visitors.

What do you think?
